src/PrestaShopBundle/Form/Admin/Product/ProductPrice.php

```diff
@@ -41,28 +52,83 @@ class ProductPrice extends CommonAbstractType
     // however the ID is required for some fields so we use a default one:
     const DEFAULT_PRODUCT_ID_FOR_FORM_CREATION = 1;
 
-    private $translator;
-    private $tax_rules;
-    private $tax_rules_rates;
+    /**
+     * @var Configuration
+     */
     private $configuration;
-    private $eco_tax_rate;
+    /**
+     * @var CountryDataProvider
+     */
+    private $countryDataprovider;
```

Comment:
I guess this code is here because, somewhere in this php class, we use `$this->countryDataprovider`

This raises a question: when a variable is declared in local scope like this, but has no dedicated statement, what is his scope ? Public, protected, private ?

So I tried ... and it seems it's public:
```
<?php

class A
{
	public function alo()
	{
		$this->a = 'aaa';
		echo $this->a.PHP_EOL;
	}
}

$b = new A();
$b->alo();
echo $b->a.PHP_EOL;
```
Output:
```
aaa
aaa
```

So if you create "missing" variable declarations for PHPStan, in order to comply with SemVer requirements, these properties need to be public 😅 

Looks dirty though ... wdyt @PierreRambaud

Reply:
I think it's better to set them private, even if it is considered a BC break (small one though)

Reply:
@matks You're right. If I can avoid a BC ;) I'll do it

Ping @PrestaShop/prestashop-core-developers

Reply:
If properties are not declared yes, it's better to  declare them as public (sadly)

Reply:
@matks Fixed ;)